docs/02.API-REFERENCE.md

```diff
@@ -4318,6 +4388,95 @@ main (void)
 - [jerry_parse_and_save_snapshot](#jerry_parse_and_save_snapshot)
 
 
+## jerry_load_function_snapshot_at
+
+**Summary**
+
+Load the selected snapshot function from the specified buffer as a function object.
```

Comment:
I would mention that the lexical env is always the global lexical env.

Reply:
@zherczeg I've updated the docs.